extensions/basic_auth/README.md

```diff
@@ -104,10 +79,13 @@ message BasicAuth {
 ```
 
 ## Feature Request and Customization
+
 ---
 
 If you have any feature request or bug report, please open an issue in this repo. Currently it is on the roadmap to:
-- [ ] Read secret from local file. This is pending on Wasm ABI supporting file read.
-- [ ] Add regex to path matching
 
-It is recommended to customize the extension according to your needs. Please take a look at [Wasm extension C++ development guide](../doc/write-a-wasm-extension-with-cpp.md) for more information about how to write your own extension.
+* [ ] Read secret from local file. This is pending on Wasm ABI supporting file read.
```

Comment:
Do we have an opening issue in WASM ABI repo?

Reply:
Created one and linked here.